Feature/moreandhide

rlacksals96 opened the pull request:
feat:
 collapse: version list들이 checkbox 체크시 나열되도록 변경. 체크해제시 숨김
 search box: hover시 그림자 활성화
add:
 footer: copyright(MIT license) 문구 추가
edit:
 logo design: font 변경

rlacksals96 committed:
feat:more and hide

rlacksals96 committed:
style:logo,search_box,footer

rlacksals96 committed:
feat:collapse

rlacksals96 committed:
feat:Collapse

always0ne commented on src/components/Selector.js:
```diff
@@ -25,24 +27,14 @@ const Selector = ({type, onSubmit}) => {
-                            </li>)
-                        }
-                    </ul>
+                    {/*<More id={index} value={content.category} changeMore={changeMore}/>*/}
```
주석처리된 코드는 어떤 용도의 코드였나요?

rlacksals96 replied:
원래 내가 만들려고 했다가 밑에 category부분이 collapse 안되서 실패했던거. 다음 push때 제거된 상태로 올라가도록 지울게

always0ne commented on src/pages/main.js:
```diff
@@ -65,8 +66,15 @@ const Main = ({type}) => {
-            <Recommend recommands={recommends}/>
-            <footer/>
+            <Recommend recommands={recommends} searchString={searchString}/>
+            <footer>
+                <div id="copyright">Copyright 2020 <b>SSU</b></div>
+                <div>
+                    Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is fur ...
+                    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+                </div>
+
+            </footer>
```
그리고 가능하면 Main은 레이아웃잡아놓는 용도로 사용하는게 좋을거 같아요
Footer 컴포넌트 생성해서 작업 들어가는게 좋을 거 같아요
그리고 현재 너무 빡빡하게 글씨가 들어가있는 느낌이여서 차라리 다른걸로 채우는게 좋을거 같네요

rlacksals96 replied:
오키 따로 컴포넌트 생성하고 #copyright 부분만 살리고 나머진 다른거 찾아볼게

always0ne replied:
그리고 저작권을  SSU로 하지 말고 차라리 팀으로 해요

always0ne commented on src/components/Selector.js:
```diff
@@ -25,24 +27,14 @@ const Selector = ({type, onSubmit}) => {
-    return (
+
+        return (
         <ol className="test">
```
choose your environments 가 리스트 최상단에 추가되는게 사용자 접근성 부분에서 더 좋을거 같아요

rlacksals96 replied:
설명으로 "choose your environment"를 추가하면 좋을거 같다는 의미지? 태그 추가할게

always0ne reviewed (commented):
그리고 추가적으로 표 디자인 어느정도 개선이 되야 할거 같아요
ex)True false가 아니라 이미지가 나오게
등등

rlacksals96 committed:
edit: layout changed

mv checkbox left side of each environment
mk Footer component, rm license explanation
add instruction "choose your environment!"

rlacksals96 committed:
edit: design

change available result: t/f -> check/cross img
footer: change to copyright 2020 reserved-words all rights reserved

rlacksals96 committed:
edit:redesign table

always0ne reviewed (approved):
고생했어요 머지합시다
